src/services/analytics/useGtm.ts

```diff
@@ -79,6 +82,10 @@ const useGtm = () => {
     gtmTrackPageview(router.pathname)
   }, [router.pathname])
 
+  useEffect(() => {
+    gtmSetUserProperty('walletLabel', walletLabel ?? 'NONE')
```

Comment:
Please create an enum for possible user properties, similar to how to define custom events.